server/controllers/bookingRequest.js

```diff
@@ -0,0 +1,81 @@
+const asyncHandler = require("express-async-handler");
+const User = require("../models/User");
+const BookingRequest = require("../models/BookingRequest");
+
+// @route POST /booking-requests/
+// @desc Create a new request
+// @access Private
+
+exports.createBookingRequest = asyncHandler(async (req, res, next) => {
+  const userId = req.user.id;
+  const user = await User.findById(userId);
+  const { sitterId, start, end, description } = req.body;
+
+  const sitter = await User.findById(sitterId);
+  if (sitter) {
+    const bookingRequest = await BookingRequest.create({
+      userId: user,
+      sitterId: sitter,
+      start,
+      end,
+      description,
+    });
+
+    res.status(201).json({
+      success: {
+        bookingRequest: {
+          request_id: bookingRequest._id,
```

Comment:
```suggestion
          requestId: bookingRequest._id,
```
Please use camelCase